traxxas_slash_4x4_platinum_2012/Dockerfile

```diff
@@ -0,0 +1,39 @@
+#  Copyright (C) 2018-2021 LEIDOS.
+# 
+#  Licensed under the Apache License, Version 2.0 (the "License"); you may not
+#  use this file except in compliance with the License. You may obtain a copy of
+#  the License at
+# 
+#  http://www.apache.org/licenses/LICENSE-2.0
+# 
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#  WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#  License for the specific language governing permissions and limitations under
+#  the License.
+
+
+# GENERIC Template for CARMA Configuration Dockerfiles
+# Do not invoke directly, symlink into configuration folders below and invoke from there
+
+FROM busybox:latest
```

Comment:
Just curious these images are used at runtime have you been able to verify they work on the f1/10 cars?

Reply:
As far as I know it's working. I haven't tried the `carma config install` script for a while because at the time the C1T images were not available under the `usdotfhwastoldev` org on Docker Hub. I have been building the `carma-config` image locally, then running `carma config set usdotfhwastoldev/carma-config:noetic-f1tenth-develop`. From there, I can launch the system with `carma start all`.

Reply:
Ok sounds good